frontend/src/app/[locale]/dev/feature-flags/FeatureFlagsTable.tsx

```diff
@@ -1,39 +1,18 @@
 "use client";
 
 import { useFeatureFlags } from "src/hooks/useFeatureFlags";
-import { useUser } from "src/services/auth/useUser";
 
 import React from "react";
 import { Button, Table } from "@trussworks/react-uswds";
 
-import Loading from "src/components/Loading";
-
 /**
  * View for managing feature flags
  */
 export default function FeatureFlagsTable() {
   const { setFeatureFlag, featureFlags } = useFeatureFlags();
-  const { user, isLoading, error } = useUser();
```

Comment:
with a working header we no longer need these changes on the feature flag page as a proof of concept